docs/dev/adding_instruments.rst

```diff
@@ -105,6 +105,13 @@ This is a minimal instrument definition:
 
 Make sure to include the PyMeasure license to each file, and add yourself as an author to the :code:`AUTHORS.txt` file.
 
+There is a certain order of elements in an instrument class that is useful to adhere to:
+
+* First, the initializer (the :code:`__init__()` method), this makes it faster to find when browsing the source code.
+* Then class attributes/variables, if you need them.
+* Then properties (pymeasure-specific or generic Python variants). This will be the bulk of the implementation.
+* Finally, any methods.
```

Comment:
It is one option, to separate the properties and methods according to their type.

I, on the other hand, prefer to sort thematically, e.g. I'd add a `start_scan()` method to the `scan_speed`, `scan_range`, and `scan_center` properties.

I guess we can leave it, how you wrote it, because these methods (like `start_scan()`) are rare and can be an exception to this rule and stay with their buddy properties.

Reply:
I see your point. I can emphasize that it's a guideline, not a strict policy. If people start thinking about the order then it's already accomplished most of what I want. :-)

Reply:
You wrote, that it is a useful order. Therefore it is marked as a suggestion. Keep it as is.